upgrade/upgrade-scripts/upgrade-container

```diff
@@ -443,6 +443,29 @@ function stop() {
 		sleep 1
 	done
 
+	#
+	# Starting the upgrade container would have enabled ip forwarding
+	# on the host (and in turn disabled LRO). Reset these settings
+	# on a best effor basis.
+	#
+	sysctl -w net.ipv4.ip_forward=0 &>/dev/null
```

Comment:
I'm good as is, but we do have the `warn` function.. so you could do something like:
```
sysctl ... || warn "..."
```
with that said, since we already emit `xtrace` to syslog, I'm not sure adding `warn` provides much benefit..

Reply:
Added the warn. Makes the code more readable even if functionally it may not add much value.